DEPRECATIONS.md

```diff
@@ -4,15 +4,15 @@ This document lists deprecated constructs in the SDK, with their expected time 
 
 | API / Feature | Deprecated (warning) | Deprecated (error) | Removed | Notes |
 | --- | --- | --- | --- | --- |
-| `ChatClient#searchMessages` <br/>*client* | 2021.09.14<br/>4.18.0 | 2021.10.01 ⌛ | 2021.11.01 ⌛ | Use the `ChatClient#searchMessages` method with unwrapped parameters instead |
-| `ChatDomain#createDistinctChannel` <br/>*offline* | 2021.09.14<br/>4.18.0 | 2021.10.14⌛ | 2021.11.14 ⌛ | Use ChatClient::createChannel directly |
-| `ChatDomain#removeMembers` <br/>*offline* | 2021.09.14<br/>4.18.0 | 2021.10.14⌛ | 2021.11.14 ⌛ | Use ChatClient::removeMembers directly |
-| `User#name` extension<br/>*client* | 2021.09.14<br/>4.18.0 | 2021.09.14<br/>4.18.0 | 2021.10.14 ⌛ | Use class member instead |
-| `User#image` extension<br/>*client* | 2021.09.14<br/>4.18.0 | 2021.09.14<br/>4.18.0 | 2021.10.14 ⌛ | Use class member instead |
-| `Channel#name` extension<br/>*client* | 2021.09.14<br/>4.18.0 | 2021.09.14<br/>4.18.0 | 2021.10.14 ⌛ | Use class member instead |
-| `Channel#image` extension<br/>*client* | 2021.09.14<br/>4.18.0 | 2021.09.14<br/>4.18.0 | 2021.10.14 ⌛ | Use class member instead |
-| `ChatClient#getMessagesWithAttachments`<br/>*client* | 2021.08.24<br/>4.17.0 | 2021.08.31 ⌛ | 2021.09.14 ⌛ | Use getMessagesWithAttachments function with types list instead |
-| `ChannelClient#getMessagesWithAttachments`<br/>*client* | 2021.08.24<br/>4.17.0 | 2021.08.31 ⌛ | 2021.09.14 ⌛ | Use getMessagesWithAttachments function with types list instead |
+| `ChatClient#searchMessages` <br/>*client* | 2021.09.15<br/>4.18.0 | 2021.10.15 ⌛ | 2021.11.15 ⌛ | Use the `ChatClient#searchMessages` method with unwrapped parameters instead |
+| `ChatDomain#createDistinctChannel` <br/>*offline* | 2021.09.15<br/>4.18.0 | 2021.10.15 ⌛ | 2021.11.15 ⌛ | Use ChatClient::createChannel directly |
+| `ChatDomain#removeMembers` <br/>*offline* | 2021.09.15<br/>4.18.0 | 2021.10.15 ⌛ | 2021.11.15 ⌛ | Use ChatClient::removeMembers directly |
+| `User#name` extension<br/>*client* | 2021.09.15<br/>4.18.0 | 2021.09.15<br/>4.18.0 | 2021.10.15 ⌛ | Use class member instead |
+| `User#image` extension<br/>*client* | 2021.09.15<br/>4.18.0 | 2021.09.15<br/>4.18.0 | 2021.10.15 ⌛ | Use class member instead |
+| `Channel#name` extension<br/>*client* | 2021.09.15<br/>4.18.0 | 2021.09.15<br/>4.18.0 | 2021.10.15 ⌛ | Use class member instead |
+| `Channel#image` extension<br/>*client* | 2021.09.15<br/>4.18.0 | 2021.09.15<br/>4.18.0 | 2021.10.15 ⌛ | Use class member instead |
+| `ChatClient#getMessagesWithAttachments`<br/>*client* | 2021.08.24<br/>4.17.0 | 2021.09.15<br/>4.18.0 | 2021.09.29 ⌛ | Use getMessagesWithAttachments function with types list instead |
```

Comment:
We adjust the future deprecation schedule by adding 14 days to the current day, right?